Composer/packages/client/src/components/Split/LeftRightSplit.tsx

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { default as Measure, ContentRect } from 'react-measure';
+
+const defaultSplitterWidth = 5;
+
+const MeasureDiv = styled.div`
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  outline: none;
+  overflow: hidden;
+`;
+
+const Root = styled.div(({ leftColWidth, splitterWidth }: { leftColWidth: string; splitterWidth: number }) => ({
```

Comment:
why one is string the other one is number that will get converted to px string value later on L27?
They both can be either number or string right?

Reply:
The pane widths can be percent or pixels, but the splitter width can only be a pixel width.  I'll think about what I might want to do to allow pixel strings for the splitter width prop - future change.